### PR TITLE
Add content negotiation to resolver

### PR DESCRIPTION
**Pull request by cthoyt**
Now, if you request

https://semantic.farm/GO:0032571?format=turtle or request https://semantic.farm/GO:0032571 with Accept mimetype text/turtle (or related RDF mimetype), then you'll get 

```turtle
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .

<https://semantic.farm/GO:0032571> rdfs:seeAlso <http://bio2rdf.org/go:0032571>,
        <http://identifiers.org/obo.go/GO:0032571>,
        <http://purl.obolibrary.org/obo/GO_0032571>,
        <http://purl.org/obo/owl/GO#GO_0032571>,
        <http://www.geneontology.org/GO:0032571>,
        <http://www.informatics.jax.org/searches/GO.cgi?id=GO:0032571>,
        <http://www.informatics.jax.org/vocab/gene_ontology/GO:0032571>,
        <http://www.pantherdb.org/panther/category.do?categoryAcc=GO:0032571>,
        <https://bioportal.bioontology.org/ontologies/GO/?p=classes&conceptid=http://purl.obolibrary.org/obo/GO_0032571>,
        <https://identifiers.org/GO:0032571>,
        <https://n2t.net/go:0032571>,
        <https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032571>,
        <https://www.ebi.ac.uk/QuickGO/term/GO:0032571>,
        <https://www.ebi.ac.uk/ols4/ontologies/go/terms?iri=http://purl.obolibrary.org/obo/GO_0032571>,
        <https://www.nextprot.org/term/GO:0032571> .
```

based on suggestion from @egonw. Still in preview - it might make sense to format this RDF differently. Also, it's not clear what kinds of CURIE normalization should be done, but that can be in a second pass.

### CODE REVIEW

File: src/bioregistry/app/components/github_blueprint.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""A blueprint for resolving GitHub issues and pull requests."""
```
Comment (egonw) [inline src/bioregistry/app/components/github_blueprint.py:1]: How is this related to the RDF functionality?

Comment (cthoyt) [reply src/bioregistry/app/components/github_blueprint.py:1]: Not related. This pr got a bit big because I needed to do some refactoring to make the new content negotiation working. I might try and split it apart into multiple PRs later

Comment (egonw) [reply src/bioregistry/app/components/github_blueprint.py:1]: Got it, clear!

Comment (cthoyt) [reply src/bioregistry/app/components/github_blueprint.py:1]: I split the unrelated refactoring into a different PR, so you can now see the specific changes for this.
